### ISSUE TITLE
feat: Emoji Integration into the quotes

### ISSUE THREAD

**Issue by shravan20** [shravan20/github-readme-quotes#238]
**Is your feature request related to a problem? Please describe.**
Currently, no support for emojis

**Describe the solution you'd like**
Provide support for emojis in the github readme quotes on the card.

**Comment by GRenziITDM** [comment]
Hi @shravan20, could you please provide some additional information? It seems that you can already use emojis in quotes (screenshot). What's the expected behavior?

![image](https://github.com/shravan20/github-readme-quotes/assets/20328554/56ed0904-56c1-4a11-bed5-ad6f6ead3227)

![image](https://github.com/shravan20/github-readme-quotes/assets/20328554/9958bd32-8a5c-49a2-8f26-ca49c6f076de)

**Comment by Adidem23** [comment]
Please assign  this issue to me

**Comment by shravan20** [comment]
>  could you please provide some additional information? It seems that you can already use emojis in quotes (screenshot). What's the expected behavior?

@GRenziITDM @Adidem23 : Want to populate emoji in existing quotes itself, as an optional bit, filter string without emoji by deafult and includes, only if enabled via query strings.

**Comment by Adidem23** [comment]
Yeah I have populated strings with emoji which were there in json file . Now you can filter strings without emoji and populate them with emojis

**Comment by Uriyo** [comment]
@Adidem23 is the issue resolved?

**Comment by Adidem23** [comment]
yes

**Comment by monushah108** [comment]
hey @zhravan  is this issue still relevent . can i work on this issue
